scipy/sparse/linalg/_eigen/_svds_doc.py

```diff
@@ -253,9 +253,11 @@ def _svds_lobpcg_doc(A, k=6, ncv=None, tol=0, which='LM', v0=None,
     True
 
     The singular vectors are also orthogonal.
+
     >>> (np.allclose(u3.T @ u3, np.eye(5)) and
     ...  np.allclose(vT3 @ vT3.T, np.eye(5)))
     True
+
```

Comment:
FYI: Technically, a blank line at the end of the docstring is not a requirement.  I originally proposed a guideline about this in https://github.com/scipy/scipy/pull/13955, but I didn't really care either way, and [feedback](https://github.com/scipy/scipy/pull/13955#discussion_r622689772) was that this was not significant enough for a strict guideline, so I removed it.  So we have no rule about this, and either way is acceptable.

There is no need to undo this change, however; I'm fine with merging it.